pkg/webhook/validating/pod.go

```diff
@@ -78,13 +78,18 @@ func PodCreateValidation(ctx context.Context, req *admission.Request, handler *H
 			}); err != nil {
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
-		if len(ipList.Items) >= 1 && ipList.Items[0].Spec.Network != specifiedNetwork {
-			return admission.Denied(fmt.Sprintf(
-				"pod has assigned ip %s of network %s, cannot assign to another network %s",
-				ipList.Items[0].Spec.Address.IP,
-				ipList.Items[0].Spec.Network,
-				specifiedNetwork,
-			))
+
+		for i := range ipList.Items {
```

Comment:
It's better to use "for _, ipInstance := range ipList.Items" instead of index only?

Reply:
this will avoid deepcopy when ranging because element in `Items` is value type not pointer type